core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeRelationService.java

```diff
@@ -33,6 +34,11 @@ public List<EnrolleeRelation> findByTargetEnrolleeId(UUID enrolleeId) {
         return dao.findByTargetEnrolleeId(enrolleeId);
     }
 
+    public boolean isUserProxyForAnyOf(UUID participantUserId, List<Enrollee> enrolleeList) {
```

Comment:
Since this method only needs the enrolleeIds, I might change the signature to just take a list of EnrolleeIds instead of full Enrollee objects